src/utils/filterTodos.ts

```diff
@@ -0,0 +1,18 @@
+import { FilterOptions } from '../types/FilterOptions';
+import { Todo } from '../types/Todo';
+
+export const filterTodos = (todos: Todo[], filter: FilterOptions) => {
+  return [...todos].filter(todo => {
```

Comment:
```suggestion
  return todos.filter(todo => {
```
`filter` creates a new array, you don't need a copy